payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/autocomplete/PlacesClientProxy.kt

```diff
@@ -39,15 +42,35 @@ interface PlacesClientProxy {
             googlePlacesApiKey: String,
             isPlacesAvailable: IsPlacesAvailable = DefaultIsPlacesAvailable(),
             clientFactory: (Context) -> PlacesClient = { Places.createClient(context) },
-            initializer: () -> Unit = { Places.initialize(context, googlePlacesApiKey) }
+            initializer: () -> Unit = { Places.initialize(context, googlePlacesApiKey) },
+        ): PlacesClientProxy {
+            return create(
+                context,
+                googlePlacesApiKey,
+                isPlacesAvailable,
+                clientFactory,
+                initializer,
+                ErrorReporter.createFallbackInstance(context)
+            )
+        }
+
+        @VisibleForTesting
```

Comment:
I had to create this as a separate constructor, because the other one is visible as part of our public API. Another option would be to change the original constructor to take an ErrorReporter and then create the ErrorReporter in AddressElementViewModelModule, but that would change the public API

Reply:
I think it's safe to update the original. We can mark the companion object as restrict to as well. We have quite a few instances where we mark the outer class, but not the inner class (in this case the inner companion object).

We don't consider this public api since the outer class is marked restrict to. This is just an oversight.

Reply:
yay! Ok updated this so that there's just one PlacesClientProxy create function and it takes in an ErrorReporter